pkg/cloudprovider/providers/azure/azure_storage.go

```diff
@@ -68,3 +68,19 @@ func (az *Cloud) DeleteFileShare(accountName, key, name string) error {
 	return nil
 
 }
+
+// ResizeFileShare resize a file share
+func (az *Cloud) ResizeFileShare(accountName, accountKey, name string, sizeGB int) error {
+	fileClient, err := az.getFileSvcClient(accountName, accountKey)
+	if err != nil {
+		return err
+	}
+
+	share := fileClient.GetShareReference(name)
+	share.Properties.Quota = sizeGB
```

Comment:
Is it GB or GiB? It appears that you are passing GiB value from other place.

Reply:
thanks for checking, it's GiB, have changed the var name